listpartitionreassignments.go

```diff
@@ -0,0 +1,135 @@
+package kafka
+
+import (
+	"context"
+	"net"
+	"time"
+
+	"github.com/segmentio/kafka-go/protocol/listpartitionreassignments"
+)
+
+// ListPartitionReassignmentsRequest is a request to the ListPartitionReassignments API.
+type ListPartitionReassignmentsRequest struct {
+	// Address of the kafka broker to send the request to.
```

Comment:
I think we need to add tags to these structs like we've done elsewhere
https://github.com/segmentio/kafka-go/blob/f48706ef46bbf987630e4b8d728fa7f1a2f831b0/protocol/alterclientquotas/alterclientquotas.go#L11